.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: cimg/rust:1.65.0
+      - image: cimg/rust:1.75.0
```

Comment:
The latest version of the `ahash` library is not compatible with Rust 1.65.0. Therefore, I've updated Rust to 1.75.0.

You can find an example of the build failure [here](https://app.circleci.com/pipelines/github/kyren/hashlink/135/workflows/e3376f60-ed9e-4f66-98d0-b19ebb5c3c52/jobs/128). This was triggered by the previous PR #24, but the issue remains the same. Without a cargo artifacts caching system to maintain a snapshot of dependencies from their initial fetch, I believe the build will also fail on the master branch.

Reply:
FWIW, that was true of ahash 0.8.8 which required 1.72, but 0.8.9 and later relaxed back to 1.60.

Reply:
Ah, awesome, thanks @cuviper for letting me know, I'll revert this line then.